packages-exp/auth-exp/src/core/auth/auth_impl.ts

```diff
@@ -271,6 +275,20 @@ export class AuthImplCompat<T extends User> implements Auth, _FirebaseService {
     return `${this.config.authDomain}:${this.config.apiKey}:${this.name}`;
   }
 
+  _startProactiveRefresh(): void {
+    this.isProactiveRefreshEnabled = true;
```

Comment:
do we need this on the auth object? or should it just live exclusively on the user?

Reply:
Auth needs to know about it because start/stop are tightly coupled with the current user lifecycle.